Cargo.toml

```diff
@@ -27,6 +27,7 @@ edition = "2018"
 alga = "0.9.2"
 arrayvec = "0.5.1"
 byteorder = "1.3.4"
+clap = { git = "https://github.com/clap-rs/clap/", rev = "5b9dbee5db540f44c336c2db54ece444624c30ef" }
```

Comment:
Why this particular revision?

Reply:
Unfortunately there is no package pushed yet to crates.
This was just one of the latest commits. I wanted to tag it to a specific version. Will adjust it, once we have a version on crates.

Reply:
Oh I see